src/data/resolvers/colony.ts

```diff
@@ -94,6 +94,11 @@ export const getProcessedColony = async (
   let avatarObject: { image: string | null } | null = { image: null };
   let tokenAddresses: Array<Address> = [];
 
+  /*
+   * @FIX: the metadata key in metadataHistory is a string, and sorted as such.
+   * Cannot therefore be used to determine the most recent metadata.
+   */
+
```

Comment:
Hmmm, can you point out where ' metadataHistory' is being ordered by ascending order? As far as I know the only problem we have with IPFS data not being recent is when IPFS fails to retrieve data

Reply:
I committed my proposed fix with the message "[Fix: correct metadata fallback in ProcessedColony resolver](https://github.com/JoinColony/colonyDapp/pull/3643/commits/b4e4a32dbd7a9cb510e5a06680e6f85dfbc09147)". If you go into the resolver and look at the metadata history array, you can see it's being ordered alphabetically per the metadata hashes:

![metadata-order](https://user-images.githubusercontent.com/64402732/180663485-2a07e01c-9aad-4e66-8aab-3600ba91aaf9.png)

My fix queries for the timestamp and then uses an existing function `sortMetadataHistory` to get it so the most recent entry is last in the array.

Reply:
Just tested it, looks good to me! At first, that you mentioned this was really weird to me because we haven't received any bug report about the metadata being weird and that code has been there for a long time, but oh well, I don't see any problems with your implementation.